application/src/pages/NavBar/NavBar.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { NavBar } from './NavBar';
+import { store } from '../../bll/store';
+import i18n from '../../utils/i18next';
+
+function setup() {
+  const { getByText, queryByText } = render(
+    <BrowserRouter>
+      <Provider store={store}>
+        <NavBar />
+      </Provider>
+    </BrowserRouter>
+  );
+  return [getByText, queryByText];
+}
+describe('NavBar component', () => {
+  beforeEach(() => {
+    i18n.init();
```

Comment:
I think you can do it without beforeEach in this case

Reply:
removed